charts/nri-kube-events/templates/deployment.yaml

```diff
@@ -79,6 +79,10 @@ spec:
             - name: config
```

Comment:
When you update a chart you need to bump the version as well :)

Reply:
It seem we recently handle chart release in a batch manner that dev does not bump chart version in single PR. 
We bump and release both app (if any app change) and helm chart weekly for all PRs merged during the past week through weekly release meeting.
So I do not bump the chart version here. cc @htroisi

Reply:
I am not sure of the correct process wrt app and chart version bumping. Let's get @vihangm to advise on our release process once he gets back from vacation.

We held off on bumping the app & chart version in your `nri-kubernetes` PR because the repo had changes on main that that we weren't ready to release. For this PR, I believe we can bump the helm chart version (which will create a helm chart release once merged).

Reply:
Thanks, I will bump a patch version in this PR then for now.